src/Selenium2Driver.php

```diff
@@ -189,14 +189,7 @@ public static function getDefaultCapabilities()
     {
         return array(
             'browserName'       => 'firefox',
-            'version'           => '9',
```

Comment:
Can we make the default `version` the empty string, which signifies the "latest" available version of the browser (or, technically, [an "unknown" version](https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities#used-by-the-selenium-server-for-browser-selection)). I think it's a better default than forcing users to specify the browser version explicitly, and it would make the change backwards compatible.

What do you think?

Reply:
is it actually mandatory to provide a version in the capabilities ? In my testing, not passing the capability does the same than allowing any value in it.

Reply:
In some versions of Selenium, at least, this is a required parameter. It might be safer to pass the empty string, as documented in the link I mentioned above.